internal/synclet/server_adapter.go

```diff
@@ -20,7 +22,17 @@ func NewGRPCServer(del *Synclet) *GRPCServer {
 var _ proto.SyncletServer = &GRPCServer{}
 
 func (s *GRPCServer) GetContainerIdForPod(ctx context.Context, req *proto.GetContainerIdForPodRequest) (*proto.GetContainerIdForPodReply, error) {
-	containerID, err := s.del.ContainerIDForPod(ctx, k8s.PodID(req.PodId))
+	name, err := reference.ParseNamed(req.ImageId)
+	if err != nil {
+		return nil, err
+	}
+
+	ref, ok := name.(reference.NamedTagged)
```

Comment:
can do this directly: https://github.com/windmilleng/tilt/blob/04f02ee6dc1ffd1407ce56857240e8d6619316cf/internal/k8s/image.go#L132

Reply:
nice!